libs/features/mas/src/price/utilities.js

```diff
@@ -29,14 +29,6 @@ const opticalPriceDivisors = {
     [Term.TWO_YEARS]: 24,
 };
 
-/**
- * Special per-currency handling rules for optical prices.
- * @type { { [currencyCode: string]: (opticalPrice: number) => number } }
- */
-const opticalPriceCurrencyRules = {
-    CHF: (opticalPrice) => Math.round(opticalPrice * 20) / 20,
-};
-
 /**
  * Creates and returns a rule for rounding optical prices based on price characteristics.
  * @param { ({ price: number, divisor: number, usePrecision: boolean }) => boolean } accept
```

Comment:
Looks like the word `accept` in the end is not needed

Reply:
`const opticalPriceRoundingRule = (accept, round) => ({ accept, round });`
`accept` is the name of the 1st parameter. How it is not needed?

Reply:
Ah yes, sorry, I did not see the wrapped part of the code, please ignore this comment